File: Activity40.c
#include <stdio.h>

/* Function prototype */
float getArea(float side);

/* Global variable to store result */
float area;

int main() {
    float side;

    printf("WELCOME TO NICO'S COMPUTING AREA OF THE SQUARE PROGRAM\n\n");

    /* Ask user input */
    printf("Input side: ");
    scanf("%f", &side);

    /* Call function */
    area = getArea(side);

    /* Display result without decimals */
    printf("Output: the area is : %.0f\n", area);

    return 0;
}

/* Function definition */
float getArea(float side) {
    return side * side;   /* Formula: S^2 */
}

File: Java/Activity40.java
import java.util.Scanner;

public class Activity40 {

    public static float getArea(float side) {
        return side * side; 
    }

    // Main function
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("WELCOME TO NICO’S COMPUTING AREA OF THE SQUARE PROGRAM");

        System.out.print("Input side: ");
        float side = scanner.nextFloat();

        float area = getArea(side); 

        System.out.printf("Output: the area is : %.2f%n", area); 
    }
}
